tests/testsuite/build_script.rs

```diff
@@ -3812,7 +3810,8 @@ fn rename_with_link_search_path() {
         &root.join("foo.dll.lib"),
         p2.root().join("foo.dll.lib"),
     ));
-    fs::remove_dir_all(p.root()).unwrap();
+    remove_dir_all(p.root())
```

Comment:
This `remove_dir_all` should at least tell us which file causes problems. It also has some special handling for `Permission Denied`, so I even have a slight hope that this'll just fix the test.

Reply:
To clarify, I am not sure which unwrap fails exactly, but looking at the output of https://ci.appveyor.com/project/rust-lang-libs/cargo/build/1.0.4613#L1554 and several similar failures, looks like it happens in `test` itself, and not in the cargo-under-test.